generators/app/templates/tests/terra-consumer-nightwatch.conf.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
If using the same webpack config as terra-core and terra-clinical, this file is not longer needed.